portfolio/src/main/webapp/script.js

```diff
@@ -50,12 +50,14 @@ function addRandomFact() {
 /**
  * Fetches message from server and adds it to DOM
  */
-function getMessage() {
-  fetch('/data')
-  .then(response => response.text())
-  .then((message) => {
-    // innerHTML is safe in this case because message comes from a static 
-    // String in the server. 
-    document.getElementById('message-container').innerHTML = message;
-  });
-}
\ No newline at end of file
+fetch('/data')
+.then(response => response.json())
+.then((message) => {
+  console.log(message[0]);
```

Comment:
I suggest removing these logs before merging. Any user of your website can see them in the dev console; they don't need to know how you debug ;)